### DISCUSSION TITLE
Welcome to js-beautify Discussions!

### DISCUSSION THREAD

**Discussion by bitwiseman** [beautify-web/js-beautify#2210] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by timothyjmorris** [comment]
Hi guys, I really like `js-beautify`! I'm using `1.15.4` for `JS` formatting and it suits all but one of my needs. I can't seem to find the right option or setting to **prevent multiple variable declarations on the same line from wrapping -- when they're initialized**. Maybe it's not a common practice, but it's my convention -- and I allow it (within reason).  

Please advise.  
### BEFORE
```
let datePart = "", year = 0, month = 0, day = 0;
let timePart = "", hours = 0, minutes = 0, seconds = 0;
```
### AFTER
```
let datePart = "",
  year = 0,
  month = 0,
  day = 0;
let timePart = "",
  hours = 0,
  minutes = 0,
  seconds = 0;
```

Yes, they're not forced to a new line when they're not initialized. So this does work. But I require initialization of all variables
```
let datePart, year, month, day;
```

Also, I'm willing to contribute if you can point me in the right direction.

thanks!

-tim
